packages/docs/pages/users/public-goods-funding/funding.mdx

```diff
@@ -0,0 +1,74 @@
+
+## Submitting the funding proposal
+
+A funding proposal can be submitted with the following CLI command:
+
+```shell copy
+namada client init-proposal \
+    --pgf-funding \
+    --data-path funding_proposal.json
+```
+
+The structure of `funding_proposal.json` is demonstrated with the following example:
+
+```json
+{
+    "proposal": {
+        "content": {
+            "title": "Fund Jonny's cryptography PhD",
+            "authors": "stewie@heliax.dev",
+            "discussions-to": "forum.namada.net/t/jonny-cryptography/1",
+            "created": "2024-01-01T00:00:01Z",
+            "license": "MIT",
+            "abstract": "Fund Jonny's cryptography PhD focusing on elliptic curves.",
+            "motivation": "Jonny is the world's most promising cryptography prodigy.",
+            "details": "It is important for humanity to support Jonny in his cryptography research.",
+        "author": "tnam1qp326qxmkncgwavskar9lfwm68x04vu8sg80crm4",
+        "voting_start_epoch": 3,
+        "voting_end_epoch": 6,
+        "activation_epoch": 12
+    },
+    "data": {
+        "continuous": [
+            {
+                "Internal": {
+                    "amount": "1000",
+                    "target": "tnam1qp326qxmkncgwavskar9lfwm68x04vu8sg80aaaa"
+                }
+            }
+        ],
+        "retro": [
+            {
+                "Internal": {
+                    "amount": "1500",
+                    "target": "tnam1qp326qxmkncgwavskar9lfwm68x04vu8sg80bbbb"
+                }
+            }
+        ]
+    }
+}
+```
+
+In the `"data"` section, both the `"continuous"` and `"retro"` fields contain a list of PGF targets.
+The `"Internal"` keyword is used to indicate on-chain funding in the native token, whereas for funding over IBC to another chain, the `"IBC"` keyword would be used.
+The attributes of an `"IBC"` object look like [this](../governance/proposal-types.mdx#ibc-pgf-proposals).
+
+The value in the `"amount"` attribute is the raw token amount in that token's denomination.
+So in the example above for a PGF funding in the native NAM token, the amount `1000` corresponds to `0.001 NAM`, since the Namada token denomination is `6`.
+
+## Voting on the funding proposal
+Voting on the funding proposal is done the same way as any other normal proposal, as described [here](../governance/governance-actions.mdx#vote-on-a-proposal).
+
+If a funding proposal was submitted by a PGF steward, then voting on the proposal is not needed, as a steward funding proposal requires no `yay` votes to pass.
+However, this kind of funding proposal can be vetoed by the community if the proposal receives more `nay` votes than `1/3` of the total active voting power.
+Additionally, if the proposal receives at least as many `nay` votes as `2/3` of the total active voting power, then the steward account will be removed from the set of PGF stewards.
```

Comment:
something is weird with the formatting here?

Reply:
thx fixed in 7f311cb7608214e1233644c708f1b722fee7efff